### PR TITLE
Up to date with m4

### PR DESCRIPTION
**Pull request by rgozim**
Build (@rgozim)
 * [x] https://github.com/ome/omero-model/pull/11
 * [x] https://github.com/ome/omero-common/pull/7
 * [x] https://github.com/ome/omero-romio/pull/4
 * [x] https://github.com/ome/omero-renderer/pull/4
 * [x] https://github.com/ome/omero-server/pull/8
 * [x] https://github.com/ome/omero-blitz/pull/13

Indexer (@mtbc)
 * [x] https://github.com/ome/omero-model/pull/14
 * [x] https://github.com/ome/omero-server/pull/10
 * [x] https://github.com/openmicroscopy/openmicroscopy/pull/5959

### CODE REVIEW

**Comment by joshmoore** [comment]
The running build for https://github.com/ome/omero-build/pull/7/commits/0a3604d6b485fb64860b5a675ca4f773eef0ff6e is an accurate representation of the decisions made this morning:

 * change groupId of ice-builder
 * hard-code omero-model hack in omero-blitz-plugin

Passes locally in docker. Possibly needs updating now to _trust_ the submodule version and only change dependencies.

cc: @sbesson @jburel @manics @rgozim @mtbc

**Comment by joshmoore** [comment]
Green! (though pointing at personal forked branches)

File: init-ci.gradle [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            maven {
+                url System.getenv("MAVEN_SNAPSHOTS_REPO_URL") ?: 'https://artifacts.openmicroscopy.org/artifactory/maven/'
+            }
+            gradlePluginPortal()
+        }
+    }
+}
+
+// Use a local m2 repo to ensure a clean build
+System.setProperty("maven.repo.local", System.getProperty("user.dir") + "/m2/repository")
```
Comment (manics) [inline init-ci.gradle]: @joshmoore / @rgozim Could you comment this line out please?

Comment (joshmoore) [reply init-ci.gradle]: Done.

**Comment by joshmoore** [comment]
Green. Using this as a verification of https://github.com/openmicroscopy/openmicroscopy/pull/5978, i.e. openmicroscopy/openmicroscopy is ready to be merged and tagged. (I'll follow the release board for that one component though.)

**Pull request merged**